specs/trust-store-trust-policy.md

```diff
@@ -95,9 +96,10 @@ Users who consume signed artifacts from OCI registries, or signed arbitrary blob
   An *object* that specifies a predefined verification level, with an option to override the Notary Project trust policy defined verification level if user wants to specify a [custom verification level](#custom-verification-level).
     - **`level`**(*string*): A REQUIRED property that specifies the verification level, supported values are `strict`, `permissive`, `audit` and `skip`. Detailed explanation of each level is present [here](#signatureverification-details).
     - **`override`**(*map of string-string*): This OPTIONAL map is used to specify a [custom verification level](#custom-verification-level).
+    - **`verifyTimestamp`**(*string*): An OPTIONAL property that specifies timestamp verification configuration. Supported values are `always` and `afterCertExpiry`. The default value is `always`, which means always verify timestamp countersignature. `afterCertExpiry` means only verify timestamp countersignature if at least one certificate in the signing certificate chain has expired at the time of verification.
```

Comment:
As per [May 20 meeting notes](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view#May-20-2024), `verifyTimestamp` should be child of `signatureVerification` and peer of `level`.

Reply:
Correct. I think this is true in the current PR?

Reply:
Resolving as discussed in 6/11/24 community meeting.